template/README.md

```diff
@@ -0,0 +1,17 @@
+## Template 
+This project includes all common dependencies and settings for kaluga subproject.
+
+#Steps for adding new subproject
+1. CopyPaste template subproject, select name for your subproject (we recommend to use only lowercase characters).
+2. `<your subproject>src/androidLibMain/AndroidManifest.xml` change `com.splendo.kaluga.template` to your subproject package name.	
+3. `<your subproject>src/androidLibAndroidTests/AndroidManifest.xml` change `com.splendo.kaluga.template` to your subproject package name.
+4. `<your subproject>src/androidLibAndroidTests/kotlin/TestActivity.kt` change package name from `com.splendo.kaluga.template` to `com.splendo.kaluga.your module`.
+5. Include your subproject to kaluga project
+	go to `kaluga/settings.gradle/kts`
+	And add in the end of the file:
+	`include(":<your subproject>")`.
+6. Add your source code. For more derails check README.md in source root and each platform folder.
+7. Remove this markdown file from your subproject.
+8. Don't forget to update copyright if files were imported from another project. To do that use **Code > Update Copyright...**.
+9. Add documentation to your code if needed.
+10. Add an example of usage to the example project.
```

Comment:
There is a step 11 (if you have instrumentation tests for Android), but I can make a PR it.

Reply:
Not sure that I understand which step. For me, it works. So yes, please make PR or let me know what can I add (Didn't research good enough instrumented tests)